scripts/fast-reboot

```diff
@@ -419,6 +428,7 @@ case "$REBOOT_TYPE" in
         sonic-db-cli STATE_DB SET "FAST_REBOOT|system" "1" "EX" "180" &>/dev/null
         ;;
     "warm-reboot")
+        check_warm_restart_in_progress
```

Comment:
Should we also add support for ignoring (forcing through) this check. Just like we have an option (`-f`) to ignore Orchagent RESTARTCHECK, if some components are not reconciled.

Reply:
Thanks for this suggestion. Ignoring the check if `-f` is selected.